models/config_access.go

```diff
@@ -134,19 +128,31 @@ func (e ConfigAccess) PK() string {
 }
 
 type UserConfigAccessSummary struct {
-	ConfigID          uuid.UUID  `json:"config_id"`
-	ConfigName        string     `json:"config_name"`
-	ConfigType        string     `json:"config_type"`
-	ExternalUserName  string     `json:"external_user_name"`
-	ExternalUserEmail string     `json:"external_user_email"`
-	CreatedAt         time.Time  `json:"created_at"`
-	DeletedAt         *time.Time `json:"deleted_at,omitempty"`
-	CreatedBy         *uuid.UUID `json:"created_by,omitempty"`
-	LastSignedInAt    *time.Time `json:"last_signed_in_at,omitempty"`
-	LastReviewedAt    *time.Time `json:"last_reviewed_at,omitempty"`
-	LastReviewedBy    *uuid.UUID `json:"last_reviewed_by,omitempty"`
+	ConfigID       uuid.UUID  `json:"config_id"`
+	ConfigName     string     `json:"config_name"`
+	ConfigType     string     `json:"config_type"`
+	Role           string     `json:"role"`
+	User           string     `json:"user"`
+	Email          string     `json:"email"`
+	CreatedAt      time.Time  `json:"created_at"`
+	DeletedAt      *time.Time `json:"deleted_at,omitempty"`
+	CreatedBy      *uuid.UUID `json:"created_by,omitempty"`
+	LastSignedInAt *time.Time `json:"last_signed_in_at,omitempty"`
+	LastReviewedAt *time.Time `json:"last_reviewed_at,omitempty"`
+	LastReviewedBy *uuid.UUID `json:"last_reviewed_by,omitempty"`
 }
 
 func (e UserConfigAccessSummary) TableName() string {
 	return "user_config_access_summary"
 }
+
+type ConfigAccessLog struct {
+	ConfigID       uuid.UUID `json:"config_id" gorm:"primaryKey"`
+	ExternalUserID uuid.UUID `json:"external_user_id" gorm:"primaryKey"`
+	ScraperID      uuid.UUID `json:"scraper_id" gorm:"primaryKey"`
```

Comment:
Why do we need scraper Id ?

We should also add `MFA bool` and `properties jsonb`

Reply:
> Why do we need scraper Id ?

tying up all config access tables to a scraper to help during cleanup/deletion.